regex-assembly/933160.ra

```diff
@@ -18,7 +18,7 @@
 ##!$ \)?\s*
 
 ##! mandatory parentheses containing optional parameters
-##!$ \(.*\)
+##!$ \([^)]*\)
```

Comment:
With this change, something like this should also trigger: `eval('runExploit()')`. The match will be against the first closing parenthesis, but that's ok. Could you add a test for that?